src/components/sectionList/translation/translationFormHooks.tsx

```diff
@@ -0,0 +1,94 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import { useState } from 'react'
+import { BaseListModel, useSchemaFromHandle } from '../../../lib'
+import { camelCaseToConstantCase } from '../../../lib/utils'
+import { Query, WrapQueryResponse } from '../../../types'
+import { WebLocale, Translation } from '../../../types/generated'
+
+type DBLocalesResponse = WrapQueryResponse<WebLocale[]>
+export interface TranslationType {
+    locale: string
+    property: string
+    value: string
+}
+
+export interface FormObj {
```

Comment:
Unfortunately we cannot define it like this, because the form-fields are dynamic. This should just be a `Record<string,string>`